setup.py

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env python
 import os
+from typing import AnyStr, Union
 from setuptools import setup, find_packages
 
 
-def read(fname):
+def read(fname: Union[bytes, os.PathLike[bytes]]) -> AnyStr:
```

Comment:
Can you please clarify why `setup.py` requires typing hints?

Reply:
> Can you please clarify why `setup.py` requires typing hints?

Sorry, I thought it might be necessary to keep consistency of using typing hints.